### ISSUE TITLE
JStree aria group is missing a name

### ISSUE THREAD

**Issue by jamesjacobs** [jadu/pulsar#1080]
SiteImprove reports the following on JStree:

![image](https://user-images.githubusercontent.com/756393/69232669-687f0e80-0b83-11ea-8229-f5a9ea290bfe.png)

Refers to where `role="group"` is used on `ul`'s

**Comment by jamesjacobs** [comment]
Partially addressed by #1263. Unable to fix SI errors. Will list details in accessibility documentation.
